client_telemetry/pcap_stats.go

```diff
@@ -42,17 +42,34 @@ func (c *PacketCounts) Copy() *PacketCounts {
 	return &copy
 }
 
+func (c *PacketCounts) CopyAndReset() *PacketCounts {
+	copy := c.Copy()
+
+	c.TCPPackets = 0
+	c.HTTPRequests = 0
+	c.HTTPResponses = 0
+	c.HTTPRequestsRateLimited = 0
+	c.OversizedWitnesses = 0
+	c.TLSHello = 0
+	c.HTTP2Prefaces = 0
+	c.QUICHandshakes = 0
+	c.Unparsed = 0
+
+	return copy
+}
+
 // Reflects the version of the JSON encoding.  Increase the minor version
 // number for backwards-compatible changes and the major number for non-
 // backwards compatible changes.
 const Version = "v0.3"
 
 type PacketCountSummary struct {
-	Version        string                   `json:"version"`
```

Comment:
Consider bumping the minor version number to signal the presence of ObservationWindow?